google_compute_engine/networking/dhcp_refresh/dhcp_refresh.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Refresh DHCP for interfaces as controlled my metadata."""
+
+import logging.handlers
+
+from google_compute_engine import logger
+from google_compute_engine.compat import distro_utils
+
+
+class DhcpRefresh(object):
```

Comment:
Rename the file `dhcp_lease_refresh` and update the function you call to be something like `HandleDhcpLease`.